modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/InfoPanelControl.js

```diff
@@ -61,6 +61,7 @@ const InfoPanelControl = ({infoPanelId, onInfoButtonClick, separator}) => {
 				onClick={onInfoButtonClick}
 				ref={infoButtonRef}
 				symbol="info-circle-open"
+				title={Liferay.Language.get('show-info-panel')}
```

Comment:
We want this to act as a tooltip not the default html title, so we need to also add the class: `lfr-portal-tooltip`.

Reply:
The language key "toggle-info-panel" would probably make more sense, since one the panel is open, the button then acts to close the panel, rather than open it.

Reply:
I renamed to "toggle-info-panel" and notice that adding the title is enough to show the tooltip, so there must be another mechanism to show the tooltip on this element, different from adding the class you mentioned. There would be a reason to add the class, even if the tooltip is already being displayed on hover?

Will send this change to frontend infra repository.